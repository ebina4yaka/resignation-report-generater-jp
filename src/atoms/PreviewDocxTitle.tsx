import React, { ReactElement } from 'react'
import { createStyles, makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles(
  createStyles({
    root: {
      textAlign: 'center',
    },
  })
)

export default function PreviewDocxTitle(): ReactElement {
  const classes = useStyles()
  return <div className={classes.root}>退職届</div>
}
