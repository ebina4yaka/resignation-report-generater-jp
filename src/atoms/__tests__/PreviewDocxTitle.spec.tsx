import React from 'react'
import renderer from 'react-test-renderer'
import PreviewDocxTitle from '../PreviewDocxTitle'

test('Snapshot test for PreviewDocxTitle', () => {
  const tree = renderer.create(<PreviewDocxTitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
