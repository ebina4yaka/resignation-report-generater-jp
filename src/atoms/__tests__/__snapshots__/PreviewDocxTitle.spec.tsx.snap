// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for PreviewDocxTitle 1`] = `
<div
  className="makeStyles-root-1"
>
  退職届
</div>
`;
